docs/agent-machine-setup-windows.md

```diff
@@ -0,0 +1,138 @@
+# Setting up a new Crater Windows agent machine
+
+## Operating System
+
+The agent machine should run Windows Server build 1803 (Windows 10 build 1803
+will **not** work, see below). The machine should have Hyper-V enabled (images
+ending in `v3` on Azure DevOps), although this may not be necessary since we
+only use process isolation. Testing has been done on a server with a GUI, but
+all installation steps can be done at the command line with the exception of
+configuring Docker Desktop. It may be possible to use a headless server if
+Docker is already properly configured. The following paragraphs contain the
+rationale for choosing this particular build of Windows. If this does not
+interest you, skip to the next section.
+
+Docker on Windows supports two different ways to run containers, process
+isolation and Hyper-V isolation. Hyper-V isolation is more secure and more
+flexible, but it is too slow for our purposes; it essentially spins up a VM for
+each container. In order to run process isolation, the build number of the
+agent machine and the docker image [need to match][ver-compat]. You can use
```

Comment:
Do you want to say "for example, if you're using Build 1803, you can find the image [here]()"

Reply:
Yeah, I'll add a link to [the official image](https://hub.docker.com/r/rustops/crates-build-env-windows).